protoutils/protoutils.go

```diff
@@ -128,8 +178,10 @@ func isEmptyValue(v reflect.Value) bool {
 	}
 }
 
-// structToMap attempts to coerce a struct into a form acceptable by grpc. Ignores omitempty tags.
-func structToMap(data interface{}) (map[string]interface{}, error) {
+// structToMap attempts to coerce a struct into a form acceptable by grpc.
+// shouldOmit specifies whether to ignore empty values if they are tagged omit empty or
```

Comment:
[nit] shouldOmit -> shouldOmitEmpty, and `tagged omit empty` -> `tagged omitempty`